### PR TITLE
Add config for the certificate agent queue polling frequency.

### PR DESCRIPTION
**Pull request by UmanShahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

This complements https://github.com/edx/edx-certificates/pull/56.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @UmanShahzad! I've created [OSPR-2198](https://openedx.atlassian.net/browse/OSPR-2198) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
+- Role: certs
+  - Added `QUEUE_POLL_FREQUENCY` to make configurable the certificate agent's queue polling frequency.
```
Comment (UmanShahzad) [inline CHANGELOG.md]: Damn, you may have wanted this to be `CERTS_QUEUE_POLL_FREQUENCY`?

Comment (clemente) [reply CHANGELOG.md]: Yes, it should be `CERTS_…` because the other one is internal.

**Comment by nedbat** [comment]
@edx/devops This looks simple.

**Review by clemente — CHANGES_REQUESTED** [review]
@UmanShahzad I think I didn't click all buttons in GitHub. Code looks fine, but yes, that variable must be renamed.

File: CHANGELOG.md (hunk 1 above)
Comment (UmanShahzad) [reply CHANGELOG.md]: :+1:

**Review by clemente — APPROVED** [review]
:+1:

- [X] I tested this: I tried this on a new server and verified that it received the setting, and that certificate_agent.py was indeed waiting for a longer time (by adding log)
- [X] I read through the code
- [X] I checked for accessibility issues
- [X] Includes documentation

**Comment by UmanShahzad** [comment]
@nedbat Can anyone take a look at this? It's a relatively simple configuration change (on this PR and the corresponding https://github.com/edx/edx-certificates/pull/56).

**Review by nedbat — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@UmanShahzad 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/4312&entry.1484655318=https://github.com/UmanShahzad&entry.752974735=2018-02-05+20:25&entry.1917517419=2018-04-27+18:22&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**